src/main/java/ml/shifu/shifu/util/Shuffler.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright [2013-2020] PayPal Software Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ml.shifu.shifu.util;
+
+/**
+ * @author Junshi Guo
+ */
+public interface Shuffler {
```

Comment:
For any interface, better to have class comments and method comments.

Reply:
Added comments.